src/codeflare_sdk/utils/generate_yaml.py

```diff
@@ -689,3 +690,38 @@ def generate_appwrapper(
     else:
         write_user_appwrapper(user_yaml, outfile)
     return outfile
+
+
+def update_pip_requirements(self):
+    pip_trusted_host = self.env.get("PIP_TRUSTED_HOST")
+    pip_index_url = self.env.get("PIP_INDEX_URL")
+    requirements_path = Path("requirements.txt")
+
+    if requirements_path.exists():
```

Comment:
Hey Christian,
I followed your verification steps and it works as expected :) Good job.

One thing is, if this is also being used to add `--trusted-host and --index-url` if they are not yet set, should we also create a `requirements.txt` file if it does not exist?